meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html

```diff
@@ -48,22 +48,20 @@
             <div class="mainbar__left">
                 <div class="image">
                     <div class="image__image">
-                        <img src="{{ project.image |thumbnail_url:'heroimage' }}" alt="{% if project.image_alt_text %}{{ project.image_alt_text }}{% else %}{% translate 'Here you can find a decorative picture.' %}{% endif %}" />
+                        <img src="{{ project.image | thumbnail_url:"heroimage" }}" alt='{% if project.image_alt_text %}{{ project.image_alt_text }}{% else %}{% translate "Here you can find a decorative picture." %}{% endif %}' width="100%" height="auto" loading="lazy"/>
```

Comment:
the linting here is a little off, I assume you'll get errors too if you make changes and try to commit but it should list all the changes you need to make, annoyingly the fix command does not deal with most of them :(

Reply:
I actually updated the linting rules to reduce them a bit so if you rebase i think it won't complain, at least not as much